### PR TITLE
worked on debugging movie form

### PR DESCRIPTION
**Pull request by fritzschoff**
resovles #3741

### CODE REVIEW

File: libs/ui/src/lib/form/display-name/display-name.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <form [formGroup]="form" fxLayout="row" fxLayoutAlign="space-between center" fxLayoutGap="24px">
   <mat-form-field appearance="outline">
-    <mat-label>First Name</mat-label>
+    <mat-label>{{ labelSuffix }}&nbsp;First Name</mat-label>
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/form/display-name/display-name.component.html:3]: Why not just putting a normal space ?

File: libs/ui/src/lib/tunnel/layout/layout.component.ts [hunk 2]
```diff
@@ -14,9 +14,10 @@ import { Router, NavigationEnd } from '@angular/router';
  * @param arithmeticOperator plus or minus
  */
 function getPage(steps: TunnelStep[], url: string, arithmeticOperator: number): string {
+
   const allRoutes = steps.map(({ routes }) => routes.map(r => r.path));
   const allPath = allRoutes.flat();
-  const current = url.split('/').pop();
+  const current = url.includes('#') ? url.split('#')[0].split('/').pop() : url.split('/').pop();
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/tunnel/layout/layout.component.ts:20]: Maybe create a small function with a good naming that do it. Here I don't understand what it does, and why.

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/ui/src/lib/form/display-name/display-name.component.html (hunk 1 above)
Comment (fritzschoff) [reply libs/ui/src/lib/form/display-name/display-name.component.html:3]: not respected in the mat-label component

File: libs/ui/src/lib/tunnel/layout/layout.component.ts (hunk 2 above)
Comment (fritzschoff) [reply libs/ui/src/lib/tunnel/layout/layout.component.ts:20]: okay, I will. If you come from summary and got a fragment in your url, the next and prev button won't show up in the tunnel layout

**Pull request merged**